app/views/unattended/provisioning_templates/finish/kickstart_default_finish.erb

```diff
@@ -61,6 +63,8 @@ fi
 
 <%= snippet "blacklist_kernel_modules" %>
 
+<%= snippet_if_exists(template_name + " custom late") %>
```

Comment:
we call this "custom post" elsewhere, could we keep that for consistency?

Reply:
Same as the previous reply: I chose "early" and "late" because finish templates don't have formal Pre and Post steps, but I will happily abide if Pre/Post is the final decision.

Reply:
Changed "custom late" to "custom post".